scripts/remove-github-action-runner.sh

```diff
@@ -2,15 +2,29 @@
 
 # This script can be used to remove a Github Action Runner on an openSUSE or SLE
 # distro. It will unconfigure the service and unregister the runner.
+# Copy the scritpt to runner:/home/<user> and run it as <user>.
+# It requires GITHUB_REPOSITORY_URL (https) and GITHUB_RUNNER_TOKEN to be set
+# e.g. export GITHUB_REPOSITORY_URL=https://github.com/epinio/epinio
+# and  export GITHUB_RUNNER_TOKEN=<current token from github settings/actions/runners/new>
+# Note: You can use the same token to add or remove multiple runners,
+#       while it will expire after 1h.
 
 set -e
 
+if [ -z "$GITHUB_REPOSITORY_URL" ]; then
+  echo "The variable GITHUB_REPOSITORY_URL is empty. Exiting"
+  exit 1
+fi
+
 if [ -z "$GITHUB_RUNNER_TOKEN" ]; then
   echo "The variable GITHUB_RUNNER_TOKEN is empty. Exiting"
   exit 1
 fi
 
+REPOSITORY_NAME=$(echo $GITHUB_REPOSITORY_URL | cut -d '/' -f 4- | sed -e 's|/$||g' -e 's|/|-|g')
```

Comment:
```suggestion
REPOSITORY_NAME=$(echo $GITHUB_REPOSITORY_URL | cut -d '/' -f 4- | sed -e 's|//*$||' -e 's|/|-|g')
```
If you want to remove multiple dashes at the end of the string.

Reply:
A bit too expensive to fully handle that here - also epinio//epinio isn't covered - in the end it's about URL checking and how action.runner config will compose service names, which we would need to match in any case then ...

Reply:
In that case you can drop the g flag, I think :)